tests/app/helpers.py

```diff
@@ -157,8 +144,8 @@ def teardown_authorization(self):
     def teardown_database(self):
         with self.app.app_context():
             db.session.remove()
-            db.drop_all()
-            db.engine.execute("drop table alembic_version")
+            for table in reversed(db.metadata.sorted_tables):
+                db.engine.execute(table.delete())
```

Comment:
As discussed in real life - if we don't delete from Frameworks then we won't need the `setup_frameworks` method at all.